loopflow-computation/src/main/java/com/farao_community/farao/loopflow_computation/LoopFlowComputation.java

```diff
@@ -192,9 +185,11 @@ private static void buildFlowFromNetwork(Network network, Crac crac, Map<Cnec, D
     }
 
     public Map<Cnec, Double> buildZeroBalanceFlowShift(Network network) {
+        if (referenceProgram == null) {
+            referenceProgram = ReferenceProgramBuilder.buildReferenceProgram(network);
+        }
```

Comment:
is this necessary with `requireNonNull(referenceProgram, "referenceProgram should not be null")` in the constructor ?

Reply:
removed